modules/viewing-project-metrics-for-distributed-workloads.adoc

```diff
@@ -0,0 +1,71 @@
+:_module-type: PROCEDURE
+
+[id="viewing-project-metrics-for-distributed-workloads_{context}"]
+= Viewing project metrics for distributed workloads
+
+[role='_abstract']
+
+In {productname-short}, you can view the following project metrics for distributed workloads:
+
+* *CPU* - The number of CPU cores that are currently being used by all distributed workloads in the selected project.
+* *Memory* - The amount of memory (GiB) that is currently being used by all distributed workloads in the selected project.
+
+You can use these metrics to monitor the progress of the distributed workloads, and to assess whether the project resources are allocated correctly.
+
+.Prerequisites
+* You have installed {productname-long}.
+
+* On the OpenShift cluster where {productname-short} is installed, user workload monitoring is enabled.
+
+* You have logged in to {productname-short}.
+ifndef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using specialized {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* Your data science project contains distributed workloads.
+
+.Procedure
+
+. In the {productname-short} left navigation pane, click *Distributed Workloads Metrics*.
+
+. From the *Project* list, select the project that contains the distributed workloads that you want to monitor.
+
+. Click the *Project metrics* tab.
+
+. Optional: From the *Refresh interval* list, select a value to specify how frequently the graphs on the metrics page are refreshed to show the latest data.
++
+You can select one of these values: *15 seconds*, *30 seconds*, *1 minute*, *5 minutes*, *15 minutes*, *30 minutes*, *1 hour*, *2 hours*, or *1 day*.
+
+. In the *Requested resources* section, review the *CPU* and *Memory* graphs to identify the resources requested by distributed workloads as follows:
++
+* Requested by the selected project
+* Requested by all projects, including the selected project and projects that you cannot access
+* Total shared quota for all projects, as provided by the cluster queue
+
++
+For each resource type (*CPU* and *Memory*), subtract the *Requested by all projects* value from the *Total shared quota* value to calculate how much of that resource quota has not been requested and is available for all projects.
+
+. Scroll down to the *Top resource-consuming distributed workloads* section to review the following graphs:
++
+* Top 5 distributed workloads that are consuming the most CPU
```

Comment:
Should this number be written as "five"?

Reply:
Not according to _IBM Style_:
"In general, use numerals to represent all numbers, however large or small."